### PR TITLE
Update protobuf-data-types.md

### PR DESCRIPTION
**Pull request by JamesNK**
Fix message and type name to be consistent.

### CODE REVIEW

**Review by shirhatti — APPROVED** [review]

File: docs/architecture/grpc-for-wcf-developers/protobuf-data-types.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ namespace CustomTypes
         public long Units { get; }
         public int Nanos { get; }
 
-        public static implicit operator decimal(CustomTypes.Decimal grpcDecimal)
+        public static implicit operator decimal(CustomTypes.DecimalValue grpcDecimal)
         {
             return grpcDecimal.Units + grpcDecimal.Nanos / NanoFactor;
         }
```
Comment (shirhatti) [inline docs/architecture/grpc-for-wcf-developers/protobuf-data-types.md:175]: Looks like you missed a few.

```diff
-        public static implicit operator CustomTypes.Decimal(decimal value)
+        public static implicit operator CustomTypes.DecimalValue(decimal value)
        {
            var units = decimal.ToInt64(value);
            var nanos = decimal.ToInt32((value - units) * NanoFactor);
-            return new CustomTypes.Decimal(units, nanos);
+            return new CustomTypes.DecimalValue(units, nanos);
        }
```

**Review by shirhatti — DISMISSED** [review]

**Comment by BillWagner** [comment]
@JamesNK @shirhatti It looks like the requested changes have been made. Is this ready for final review / merge?

**Comment by JamesNK** [comment]
Yes

**Review by BillWagner — APPROVED** [review]
Approving based on previous comments and updates.

I'll :shipit: now.

**Pull request merged**